docker-compose.yml

```diff
@@ -89,8 +89,7 @@ services:
     environment:
       - ASPNETCORE_ENVIRONMENT=Development
       - ASPNETCORE_URLS=http://+:5005
-      - ServiceRepositorySettings:RepositoryLocation=/AltinnRuntime/Repos/
-      - ServiceRepositorySettings:RuntimeAPIEndPoint=http://altinn-designer:5000/
```

Comment:
So i assume we have no requirement for  RuntimeAPIEndpoint (altinn-designer) any more

Reply:
There is for now no longer any reason for runtime to talk to the designer so is removed